lib/easyify.js

```diff
@@ -1,4 +1,4 @@
-const { isFeedType } = require('ssb-ref')
```

Comment:
NOTE: we decided that only the classic feed will use the @ + suffix signature.
so this "isFeedType" from ssb-ref is now useless.
I am not changing that in this PR because I'm already 2 levels deep in updating this and going to a third level and waiting for PRs and input will mean I won't be able to proceed (I have v limited time atm)